src/styles/blocks/card.scss

```diff
@@ -0,0 +1,114 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  width: $card-width;
+  height: $card-height;
+  border-radius: $border-radius;
+  border: 1px solid $border-color;
+
+  &__product-image {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 40px;
+  }
+
+  &__product-details {
+    width: $card-content-width-adjustment;
+    margin-inline: $padding-inline;
+    margin-bottom: $margin-bottom;
+
+    &-title {
+      height: 36px;
```

Comment:
The height for the element '.product-card__product-details-title' is set explicitly which might lead to issues if the text content overflows. Consider using `min-height` or padding to handle the content size dynamically.